build-tools/packages/build-infrastructure/src/packageManagers.ts

```diff
@@ -35,22 +35,21 @@ export class PackageManager implements IPackageManager {
 		}
 	}
 
-	public installCommand(updateLockfile: boolean): string {
+	public installCommandArgs(updateLockfile: boolean): string[] {
```

Comment:
Thoughts? My first read of the other name made me think `install` shouldn't be part of the return value.

Also, the yarn case seems to be incorrect, it should still return "install", right?
```suggestion
	public installCommandWithArgs(updateLockfile: boolean): string[] {
```

Reply:
> Also, the yarn case seems to be incorrect, it should still return "install", right?

Yeah, incomplete refactor.